api/src/types/api.ts

```diff
@@ -53,6 +53,8 @@ export interface Paper {
   year: number | null;
   influentialCitationCount?: number;
   citationVelocity?: number;
+  inboundCitations: number;
+  outboundCitations: number;
```

Comment:
Just as you did with the otherside, maybe it's worth adding a comment about what these map to? `// numCiting` etc.